notebooks/Field Apps Deployment Using Python.ipynb

```diff
@@ -0,0 +1,545 @@
+{
```

Comment:
<p>Let's remember to set the org and username to template values before merging.</p><p> </p><p>Should there be an if statement like:</p><pre class="ql-syntax" spellcheck="false">if status == "disabled":
    gis.admin.location_tracking.enable()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/field-maps-scripts/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>